Assets/MixedRealityToolkit/_Core/Utilities/Lines/Renderers/MixedRealityLineRenderer.cs.meta

```diff
@@ -3,7 +3,10 @@ guid: 1287d4d138a242f794bcfc01354d3ae2
 MonoImporter:
   externalObjects: {}
   serializedVersion: 2
-  defaultReferences: []
+  defaultReferences:
+  - lineDataSource: {instanceID: 0}
+  - lineMaterial: {fileID: 10306, guid: 0000000000000000f000000000000000, type: 0}
```

Comment:
This is an odd change

Reply:
This is adding the Unity-default line material to the MixedRealityLineRenderer class. It allows the component to always have a defaulted material.

Currently the behavior of that component is if it is on or added to an object and the material is broken or unassigned, it IMMEDIATELY disables the entire object, which is a non-intuitive editor behavior that is atypical of anything else in Unity.

This change does not prevent users from assigning a different material, it just improves the default use case.